cabal.project

```diff
@@ -53,3 +53,12 @@ if impl(ghc >= 9.10)
   constraints:
     -- Earlier versions do not compile with ghc-9.10
     , plutus-ledger-api ^>=1.31
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/fs-sim
+  tag: 984e07de44c3aa42f3720b18aa6307d83851b165
+  --sha256: 0kfhmdmyy8gqxzbrsm5ayvhv9hbiacvwfh36l4liw2nwjfp8ipz5
+  subdir:
+    fs-api
+    fs-sim
```

Comment:
I'll remove this once we the Hackage versions are downloadable -- currently, the packages are not yet visible